src/scales/scale.radialLinear.js

```diff
@@ -267,7 +266,8 @@ module.exports = function(Chart) {
 				var pointLabelPosition = scale.getPointPosition(i, outerDistance + 5);
 
 				// Keep this in loop since we may support array properties here
-				var pointLabelFontColor = valueOrDefault(pointLabelOpts.fontColor, globalDefaults.defaultFontColor);
+				var pointLabelFontColor = helpers.valueAtIndexOrDefault(pointLabelOpts.fontColor, i, globalDefaults.defaultFontColor);
```

Comment:
This is probably not the approach we'd like to take for supporting multiple colors. See https://github.com/chartjs/Chart.js/pull/5153#issuecomment-362176361

I would strongly recommend any support for multiple colors be done in another PR rather than addressing two issues in one PR